src/h265_common.cc

```diff
@@ -38,6 +38,32 @@ bool IsSliceSegment(uint32_t nal_unit_type) {
   return false;
 }
 
+bool IsSpecifiedNonVclUnit(uint32_t nal_unit_type) {
```

Comment:
suggest renaming to a positive name:

     bool IsNalUnitTypeAVcl(uint32_t nal_unit_type) {
       ...